components/dropdown/dropdown-content-styles.js

```diff
@@ -152,6 +152,7 @@ export const dropdownContentStyles = css`
 		box-sizing: border-box;
 		max-width: 100%;
 		outline: none;
+		overscroll-behavior: contain;
```

Comment:
From testing, this seems to "just work" and seems pretty low-risk... so maybe doesn't need a feature flag? Let me know if others disagree.